Gemfile.lock

```diff
@@ -1,35 +1,45 @@
 PATH
   remote: .
   specs:
-    nxt_http_client (2.0.0)
+    nxt_http_client (2.0.1)
```

Comment:
I wonder why are we bumping the gem version? We only change deploy script here, it should affect the gem itself.

Reply:
Due to released changes in #169

Reply:
Ah, missed this. Thx!